CMakeLists.txt

```diff
@@ -237,6 +237,7 @@ endif()
 
 if(USE_TENSORRT)
   message(STATUS "Using TensorRT")
+  execute_process(COMMAND bash tools/dependencies/build_onnx_trt.sh)
```

Comment:
Could you clarify why an external script is needed here? Usually we can use cmake `add_subdirectory(path EXCLUDE_ALL)`. Both onnx and onnx-tensorrt use cmake, and we can import their targets via `add_subdirectory` and then specify them as dependency for the mxnet target? The latter may be more robust

Reply:
Not sure if it's needed and why it was done like that (the original author left NVIDIA), let me try with `add_subdirectory`.